### PR TITLE
Link Form Field Filters

### PR DESCRIPTION
**Pull request by AlexGStapleton**
This PR documents `siteorigin_widgets_search_posts_results` and `siteorigin_widgets_search_posts_order_by`

### CODE REVIEW

File: widgets-bundle/form-building/link-form-field-filters.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Link Form Field Filters
+The Link form field has two filters you can use to help the results return by the field.
```
Comment (Misplon) [inline widgets-bundle/form-building/link-form-field-filters.md:2]: Quick look at this sentence. "returned by the field." perhaps?

File: widgets-bundle/form-building/link-form-field-filters.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+add_filter( 'siteorigin_widgets_search_posts_results' function( $results ) {
+	foreach ( $results as $id => $result ) {
+		// Your code here.
+	}
+	return $results;
+} );
+```
+
+### Filter siteorigin_widgets_search_posts_order_by
+This filter allows you to alter the ORDER BY for the query to the database. Default value is `post_modified DESC`.
```
Comment (Misplon) [inline widgets-bundle/form-building/link-form-field-filters.md:17]: Quick look here. Can we remove "for the" and still have the sentence work? Open to other changes.

**Comment by Misplon** [comment]
Sorry mate, I didn't check the merge order required here.

**Comment by AlexGStapleton** [comment]
> Sorry mate, I didn't check the merge order required here.

All good. 👍

**Review by Misplon — APPROVED** [review]

**Pull request merged**